src/map/clif.h

```diff
@@ -574,6 +576,19 @@ enum CZ_CONFIG {
 	CZ_CONFIG_PET_AUTOFEEDING        = 2,
 	CZ_CONFIG_HOMUNCULUS_AUTOFEEDING = 3,
 };
+/**
+* Client UI types
+**/
+enum ui_types {
```

Comment:
this enum probably better use in function where it belong
i mean replace ``int32 UIType`` to ``enum ui_types UIType``